lib/src/backend.rs

```diff
@@ -331,6 +333,7 @@ pub fn make_root_commit(root_change_id: ChangeId, empty_tree_id: TreeId) -> Comm
     }
 }
 
+/// Defines the interface for backends that can be used to store a repository.
```

Comment:
nit: Since we have several kinds of backends, say "for commit backends" here

Reply:
Good point, thanks!